### DISCUSSION TITLE
New Scratch Website with Scratch Addons Features

### DISCUSSION THREAD

**Discussion by zoraezimran** [ScratchAddons/ScratchAddons#2109] [Ideas] [answered]
There should be a scratch website but with the scratch addons features because then people won't have to download the extension and it might be hard for some so it is better to make a scratch website with the scratch addons features and you can keep on updating it with the new features you add to the normal extension or at least make a scratch website and keep the extension and keep on updating both with the new scratch addons features.

**Comment by cobaltt7** [comment, marked as answer]
Impossible unless it doesn’t integrate with the main Scratch website.

However, most editor addons are available on Turbowarp: https://tubowarp.org

Open the editor, then click “Addons” in the header.

**Reply by zoraezimran** [reply]
But not all the features are available on it and the features don't get updates plus it not like scratch much

**Comment by WorldLanguages** [comment]
It's not that easy

**Reply by zoraezimran** [reply]
why isn't it easy ?

**Comment by TheColaber** [comment]
TurboWarp is your best bet for a ScratchAddons Scratch. I don't think installing from webstore is too hard...

**Reply by CST1229** [reply]
But sometimes schools disable installing extensions.

**Reply by retronbv** [reply]
That's why you should use Turbowarp,no extensions needed

**Reply by TheColaber** [reply]
> But sometimes schools disable installing extensions.

Why would you use scratch at school... I can think of some reasons, but that's not a very strong point...

**Reply by xItzDiamondx** [reply]
I don't know, maybe you have a programming class or something?

**Reply by zoraezimran** [reply]
https://github.com/ScratchAddons/ScratchAddons/discussions/2109#discussioncomment-586206

yep

**Reply by zoraezimran** [reply]
> I don't know, maybe you have a programming class or something?

yes definitely

**Reply by zoraezimran** [reply]
> That's why you should use Turbowarp,no extensions needed

I don't know its features don't get updated and its not like scratch much
